clients/kafka-common/src/main/java/org/eclipse/hono/client/kafka/consumer/KafkaConsumerConfigProperties.java

```diff
@@ -35,12 +35,12 @@
     /**
      * The default amount of time (milliseconds) to wait for records when polling records.
      */
-    public static final long DEFAULT_POLL_TIMEOUT = 100L; // ms
+    public static final long DEFAULT_POLL_TIMEOUT_MILLIS = 1000L; // ms
 
     private final Class<? extends Deserializer<?>> keyDeserializerClass;
     private final Class<? extends Deserializer<?>> valueDeserializerClass;
 
-    private long pollTimeout = DEFAULT_POLL_TIMEOUT;
+    private long pollTimeout = DEFAULT_POLL_TIMEOUT_MILLIS;
```

Comment:
why don't we change the type of this property to Duration and save all the conversion in the consumers

Reply:
@calohmn any thoughts?

Reply:
Looking at the various configuration properties in Hono where timeouts are configured, it seems that the majority of them expect just the numeric milliseconds value instead of requiring the format used by `java.time.Duration` (see #3035).

Usage of the Duration type here would mean that a value of 800 milliseconds would have to be written as "PT0.8S" ("0.8s" would also work). The intuitive form of "800ms" wouldn't work. Therefore I tend to prefer the `long` type here, requiring the number of milliseconds.

In general, I see that this PR is missing mentioning this property in the documentation. I'll add that.

Reply:
do not forget to add a corresponding release notes entry